src/panels/lovelace/cards/hui-weather-forecast-card.ts

```diff
@@ -166,19 +167,13 @@ class HuiWeatherForecastCard extends LitElement implements LovelaceCard {
       >
         <div class="content">
           <div class="icon-image">
-            ${stateObj.state in weatherImages
-              ? html`
-                  <img
-                    class="weather-image"
-                    src="${weatherImages[stateObj.state]}"
-                  />
-                `
-              : html`
-                  <ha-icon
-                    class="weather-icon"
-                    .icon=${weatherIcons[stateObj.state] || stateIcon(stateObj)}
-                  ></ha-icon>
-                `}
+            ${weatherStateIcon ||
+            html`
```

Comment:
This fallback is already in `getWeatherStateIcon`

Reply:
It isn't. This is the main state icon. If no icon found use the state Icon